src/App.tsx

```diff
@@ -45,7 +47,7 @@ class App extends Component {
 
   configErrorHandler = (error: Error) => {
     this.setState({ error });
-    console.info('Error while loading the config.json:', error);
+    console.error('Error while loading the config.json:', error);
```

Comment:
Suggest using logDev util here and considering something else in production

Reply:
`logDev` uses info method to show the message. Do we need an alternative function to show the error messages? We can also have one and just pass a message type as a param.

Reply:
I agree, I just wanted to suggest to differentiate info and error messages for dev environment, for example:

```
  const logDev = (message, type = 'info') => {
      console[type](message);
  }
```